src/genie/libs/parser/sros/show_service_sdp_using.py

```diff
@@ -0,0 +1,94 @@
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import Any
+import re
+
+# ======================================================
+# Schema for 'show service sdp-using'
+# ======================================================
+
+
+class ShowServiceSdpUsingSchema(MetaParser):
+    """schema for show service sdp-using"""
+    schema = {
+        'total': int,
+        'sdp': {
+            Any(): {
+                'service_id': int,
+                'type': str,
+                'far_end': str,
+                'oper_state': str,
+                'ingress_label': int,
+                'egress_label': int,
+            }
+        }
+    }
+
+
+class ShowServiceSdpUsing(ShowServiceSdpUsingSchema):
+    """ Parser for show service sdp-using"""
+    cli_command = 'show service sdp-using'
+
+    def cli(self, output=None):
+
+        if output is None:
+            out = self.device.execute(self.cli_command)
+        else:
+            out = output
```

Comment:
```suggestion
        if output is None:
            output = self.device.execute(self.cli_command)
```

then on line 57 you can do
``` python
for line in output.splitlines():

Reply:
Hi Daniel,
thansk for the review - I commited the suggested changes and found during testing an additional cornercase, therefor I had to fix the parser (change label to 'str' as in 'None' label ist a valid output) and extended the golden output testfiles
thanks and regards
Michael